composer.json

```diff
@@ -9,17 +9,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "webmozart/console": "^1.0-beta5",
+        "php": ">=7.0",
```

Comment:
`~7.0.0 || ~7.1.0`? PHP is not semver and we don't have a guarantee it will work in PHP 7.2 or PHP 8. Although maybe `^7.0` is enough...